catkin_ws/src/planner/src/missions.py

```diff
@@ -237,17 +239,30 @@ def octagon(
 
         smach.StateMachine.add(
             first_state_name,
-            LinearSearch(
-                self.control, self.mapping, target_class="Octagon Table", min_objects=1
+            NavigatePinger(
+                self.control, self.mapping, self.state,
             ),
```

Comment:
octagon shouldn't have navigate pinger. Just assume that you are close enough to the octagon that you can do a regular search